src/main/java/az/edu/turing/domain/entity/PassengerEntity.java

```diff
@@ -0,0 +1,63 @@
+package az.edu.turing.domain.entity;
+
+import java.util.Objects;
+import java.util.UUID;
+
+public class PassengerEntity {
+
+    private String id;
+    private String name;
+    private String surname;
+
+    public PassengerEntity(String name, String surname) {
+        this.id = UUID.randomUUID().toString().substring(0, 8);
+        this.name = name;
+        this.surname = surname;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
```

Comment:
The setter for 'id' is not needed because 'id' is generated once using UUID and should not be modified. You can remove this method to ensure 'id' remains immutable.